tests/foreman/ui/test_host.py

```diff
@@ -2597,3 +2613,40 @@ def test_positive_remove_role_in_new_ui(self):
         up on the UI
 
     """
+
+
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_match('8')
```

Comment:
Any specific reason we are testing for rhel8 and not rhel7 and rhel9 ?

Reply:
No, this was just an effort to limit the scope of the test to not take too long to run by being parametrized 5 or 6 more times. It shouldn't matter what version of Rhel the host is on, the test doesn't care, it just needs to be able to install katello-tracer

Reply:
In that case, I think we usually parametrize the tests with all the possible versions.

Reply:
ack, if I remove this decorator it will run on all possible versions, as well as FIPS versions.